### PR TITLE
Sharing tsconfig across packages

### PR DESCRIPTION
**Pull request by oveddan**

### CODE REVIEW

**Comment by oveddan** [comment]
Current dependencies on/for this PR:
* main
  * **PR #359** <a href="https://app.graphite.dev/github/pr/ourzora/zora-protocol/359?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/ourzora/zora-protocol/359?utm_source=stack-comment).

File: packages/1155-contracts/tsconfig.json [hunk 1]
```diff
@@ -1,23 +1,7 @@
 {
+  "extends": "tsconfig/tsconfig.json",
```
Comment (iainnash) [inline packages/1155-contracts/tsconfig.json:2]: would it be smart to call this "@zoralabs/tsconfig" even it it's private in the case there is a package collision?

File: packages/tsconfig/package.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "tsconfig",
```
Comment (iainnash) [inline packages/tsconfig/package.json:2]: thoughts on `@zoralabs/tsconfig` even if it's never published would prevent collisions.

File: packages/1155-contracts/tsconfig.json (hunk 1 above)
Comment (oveddan) [reply packages/1155-contracts/tsconfig.json:2]: not a bad idea - also would improve understandability of what the import references.  will do.

Comment (iainnash) [reply packages/1155-contracts/tsconfig.json:2]: 👍

Comment (oveddan) [reply packages/1155-contracts/tsconfig.json]: done

**Review by iainnash — APPROVED** [review]
great!

**Pull request merged**